vcd/datasource_vcd_rights_container2_test.go

```diff
@@ -0,0 +1,308 @@
+//go:build role || ALL || functional
+
+package vcd
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+// This file contains the improved version of TestAccVcdRightsContainers.
```

Comment:
Do we still need the old version?

Reply:
Up to the team to decide. The old version takes about 30 seconds to run. If we want to keep it, it is not a big dent into the total running time.

Reply:
I was going to ask the same 😅 .
I'd vote to remove it as the new one is testing a "superset" of what the old one tested, so it is a bit redundant IMHO. But as the comment is quite self-explanatory and the old one is quick to run, I won't complain if we decide to keep it.

Reply:
I think so aswell, it tests the same functionality, I'd vote for removing it.

Reply:
Removed